src/app/rpmostree-pkg-builtins.c

```diff
@@ -40,17 +42,15 @@ static GOptionEntry option_entries[] = {
   { NULL }
 };
 
-static GVariant *
-get_args_variant (GVariant *handles)
-{
-  GVariantDict dict;
-  g_variant_dict_init (&dict, NULL);
-  g_variant_dict_insert (&dict, "reboot", "b", opt_reboot);
-  g_variant_dict_insert (&dict, "dry-run", "b", opt_dry_run);
-  if (handles != NULL)
-    g_variant_dict_insert_value (&dict, "install-local-packages", handles);
-  return g_variant_dict_end (&dict);
-}
+static GOptionEntry install_option_entry[] = {
+  { "install", 0, 0, G_OPTION_ARG_STRING_ARRAY, &opt_install, "Install a package", "PKG" },
```

Comment:
Interesting!  I forgot about that usage.  We should probably do that in other places.